client/app/components/sections/Topbar/Topbar.story.js

```diff
@@ -120,17 +121,60 @@ const loggedOut = (props) => ({
 
 const storifyTopbar = (props) => r(storify(r(Topbar, props)), containerStyle);
 
+const topbarWithSpecs = (props, spec) => {
+  const component = r(Topbar, props);
+  const mounted = mount(component);
+  spec(mounted);
+  return r(storify(component, containerStyle));
+};
+
+const noLoginLinks = (component) => {
+  it('shouldn\'t contain login and signup links', () => {
+    expect(component.text()).to.not.contain('login');
+    expect(component.text()).to.not.contain('signup');
+  });
+  it('should contain logout link', () => {
+    expect(component.text()).to.contain('Log out');
+  });
+};
+
+const hasLoginLinks = (component) => {
+  it('should contain login and signup links', () => {
+    expect(component.text()).to.contain('Log in');
+    expect(component.text()).to.contain('Sign up');
+  });
+  it('shouldn\'t contain logout link', () => {
+    expect(component.text()).to.not.contain('Log out');
+  });
+};
+
+const hasLogo = (component) => {
+  it('should contain logo', () => {
+    expect(component.find('.Logo')).to.exist;
```

Comment:
`eslint` fails here, see https://github.com/eslint/eslint/issues/2102

The solution seems to be either disabling `no-unused-expressions` rule, or https://github.com/prodatakey/dirty-chai. Not sure which I prefer.